src/Project/Sugcon2024/Sugcon/src/components/Events/Agenda.tsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import InnerHTML from 'dangerously-set-html-content';
-import { Field } from '@sitecore-jss/sitecore-jss-nextjs';
+import { Field, withDatasourceCheck } from '@sitecore-jss/sitecore-jss-nextjs';
 import useSWR from 'swr';
+import { ComponentProps } from 'lib/component-props';
 
 interface Fields {
   SessionizeUrl: Field<string>;
 }
 
-type AgendaProps = {
-  params: { [key: string]: string };
+type AgendaProps = ComponentProps & {
```

Comment:
Adding the ComponentProps here, introduces errors in the related StoryBook assets, can these be updated as well?

Reply:
Updated.